src/app/qgisapp.cpp

```diff
@@ -13061,6 +13062,12 @@ void QgisApp::zoomToBookmarkIndex( const QModelIndex &index )
   mBookMarksDockWidget->zoomToBookmarkIndex( index );
 }
 
+void QgisApp::takeAppScreenShots( const QString &saveDirectory, const int categories )
+{
+  QgsAppScreenShots ass( saveDirectory );
+  ass.takeScreenshots( QgsAppScreenShots::Categories( categories ) );
```

Comment:
Can you rename "takeScreenshots" to "takePicturesOf" ? ;)

Reply:
fixed